src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -497,11 +496,23 @@ ov::AnyMap get_baseline_common_config() {
         { "NPUW_SLICE_OUT", "YES" },
         { "NPUW_FUNCALL_ASYNC", "YES" }
     };
+    if (enable_compiler_dq) {
+        config.emplace("NPUW_DQ", "YES");
+        config.emplace("NPUW_DQ_FULL", "NO");
+        config.emplace("NPU_COMPILER_DYNAMIC_QUANTIZATION", true);
```

Comment:
"YES", otherwise all our past "YES" or "NO" should be `true` or `false`

Reply:
Done